scalableminds/webknossos
# Allow dynamic change of natively rendered layer (by re-applying transforms)

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- open [C555_DIAMOND_2f-TPS](https://toggletransforms.webknossos.xyz/datasets/sample_organization/C555_DIAMOND_2f-TPS/view#500,450,321,0,0.621) on the dev instance
- use the new transform-toggle-buttons next to the layers
- the position and zoom should adapt when using the toggle button so that one stays roughly in the same area
- create an annotation
- volume annotation should only be possible when the volume layer is not transformed

### TODOs:
- [x] test TPS and affine chaining further
- [x] adapt position when toggling transforms
- [x] adapt zoom when toggling transforms ?
- [x] todo-comments / refactor source/target name mismatch

### Issues:
- contributes to #[6926](https://github.com/scalableminds/webknossos/issues/6926)

------
(Please delete unneeded items, merge only when none are left open)
